internal/zinc-benchmarks/src/main/scala/xsbt/ShapelessBenchmark.scala

```diff
@@ -0,0 +1,48 @@
+package xsbt
+
+import java.util.concurrent.TimeUnit
+import org.openjdk.jmh.annotations._
+
+class ShapelessBenchmark extends BenchmarkBase {
+  _project = BenchmarkProjects.Shapeless
+}
```

Comment:
My preference would be to pass in "shapeless" as a JMH parameter to avoid the boilerplate in the file for further programs.